tyk-pro/templates/deployment-mdcb.yaml

```diff
@@ -89,18 +87,33 @@ spec:
               secretKeyRef:
                 name: {{ if .Values.secrets.useSecretName }} {{ .Values.secrets.useSecretName }} {{ else }} secrets-{{ include "tyk-pro.fullname" . }} {{ end}}
                 key: MDCBLicense
+          # Default Settings
+          - name: TYK_MDCB_SERVEROPTIONS_MINVERSION
```

Comment:
According to[ tyk docs](https://tyk.io/docs/tyk-multi-data-centre/mdcb-configuration-options/#server_optionsmin_version), should it be `TYK_MDCB_SERVEROPTIONS.MINVERSION` instead?

Reply:
I have asked Sredny about this. Dot in the environment variable feels odd. So maybe docs are wrong.

Reply:
Indeed tyk-docs were wrong.